go.mod

```diff
@@ -116,6 +116,7 @@ require (
 	gopkg.in/square/go-jose.v2 v2.5.1
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
+	xorm.io/builder v0.3.6
```

Comment:
This is actually imported in alertmanager.go (ngalert), so it should not be an implicit dependency as it was before. `go mod tidy` fixed that.